google-cloud-bigquery/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://mojo.codehaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <className>com/google/cloud/bigquery/QueryParameterValue</className>
+    <method>java.util.Map getStructTypes()</method>
+    <differenceType>7013</differenceType>
```

Comment:
do we have to add an abstract method? I think it may not be worth doing a major bump for this change. can we add a regular method w default implementation returning "this"? thank you!